### ISSUE TITLE
dependencies missing in environment

### ISSUE THREAD

**Issue by sjsrey** [spatialucr/osnap#9]
We need to keep the [environment.yml](https://github.com/spatialucr/osnap/blob/master/environment.yml) updated for dependencies so devs have a unified env.
```
ImportError: Unable to find a usable engine; tried using: 'pyarrow', 'fastparquet'.
pyarrow or fastparquet is required for parquet support
```

If the install needs more than conda, we should document that as well.

**Comment by knaaptime** [comment]
ah, i get confused about which is authoritative. It's in requirements.txt but not environment.yml

**Comment by knaaptime** [comment]
i think this was resolved with 88000b642a5524ba30e9ad6d18f87a5c7e80fd85
